compat/maven-model-builder/src/main/java/org/apache/maven/model/building/ModelProblemUtils.java

```diff
@@ -98,15 +98,10 @@ static String toId(Model model) {
      * @return The user-friendly artifact id, never {@code null}.
      */
     static String toId(String groupId, String artifactId, String version) {
-        StringBuilder buffer = new StringBuilder(128);
```

Comment:
test gap as green.

Reply:
why the double trouble ?
<img width="1172" alt="image" src="https://github.com/user-attachments/assets/03618b2b-7503-486e-a25a-f52087a8f055" />

Reply:
<img width="1075" alt="image" src="https://github.com/user-attachments/assets/16d1a701-7207-4c70-9f4f-dd72a98ed4e3" />